hivelink-core/src/test/java/org/apache/iceberg/hivelink/core/TestHiveMetadataPreservingTableOperations.java

```diff
@@ -47,9 +47,9 @@ public void testFixMismatchedSchema() {
 
     long currentTimeMillis = System.currentTimeMillis();
     StorageDescriptor storageDescriptor = new StorageDescriptor();
-    FieldSchema field1 = new FieldSchema("name", "string", null);
-    FieldSchema field2 = new FieldSchema("id", "int", null);
-    FieldSchema field3 = new FieldSchema("nested", "struct<field1:string,field2:string>", null);
+    FieldSchema field1 = new FieldSchema("name", "string", "");
```

Comment:
Could you please explain a bit why we need to make changes to these comment strings? I'm losing the context a bit.

Reply:
I think it was just that converting avro->hive using iceberg ended up setting the comment to null, while doing it through the hive-serde library ended up setting it to empty string